Automattic/wp-calypso
# Refactor the upsell flow to wait for decline button to be present

#### Changes proposed in this Pull Request

- refactor `UpsellPage.declineOffer` function to use named CSS selector and add waits to guard.
- update test step `Can search for a blog name` to search for the `blogName` + .com TLD instead of just the `blogName` as the .com TLD which is used in the test is not always the default suggestion.

#### Testing instructions

Run tests on CircleCI.

Check for the following test:
`Adding a domain to an existing site `
`Calypso Gutenberg Editor: Checkout on` - subtest accesses the upsell page.
